doc/howtos/company.rst

```diff
@@ -2,224 +2,181 @@
+hidden from the view. Indeed, the company is usually hidden if the user does not have access to
+multiple companies (i.e. when the user does not have the group `base.group_multi_company`).
 
 .. code-block:: python
 
    from odoo import api, fields, models
 
    class Record(models.Model):
-      _name = "record.restricted"
-      _check_company_auto = True
-
-      info = fields.Text()
-      company_id = fields.Many2one(
-          "res.company", string="Company", required=True, index=True,
-          default=lambda self: self.env.company,
-      )
-      user_id = fields.Many2one(
-          "res.users", string="User", required=True,
-          default=lambda self: self.env.user,
-      )
-
-      other_record_id = fields.Many2one("other.record", check_company=True)
-
-No specific new feature, do notice that when using a required `company_id` field,
-a default to the current company should be set.  It ensures an easier flow to
-the user and ensures the field is set even if the user doesn't see the field (
-which is generally the case in Odoo, we only show the multi-company fields when
-user has access to multiple companies, i.e. has the group `base.group_multi_company`).
+       _name = 'record.restricted'
+       _check_company_auto = True
+
+       company_id = fields.Many2one(
+           'res.company', required=True, default=lambda self: self.env.company
+       )
+       other_record_id = fields.Many2one('other.record', check_company=True)
+
 
 .. _howto/company/views:
 
 Views
 -----
 
-As said above, when defining views, we specify the `company_id` fieds as visible
-only when user has access group `base.group_multi_company`.  This simplifies the views
-for unpriviledged users.
+As stated in :ref:`Default company <howto/company/default_company>`, the company is usually hidden
+from the view if the user does not have access to multiple companies. This is tested with the group
+`base.group_multi_company`.
 
 .. code-block:: xml
 
-  <record model="ir.ui.view" id="record_form_view">
-    <field name="name">record.restricted.form</field>
-    <field name="model">record.restricted</field>
-    <field name="arch" type="xml">
-        <form>
-            <sheet>
-                <group>
-                    <group>
-                        <field name="info"/>
-                        <field name="company_id" groups="base.group_multi_company"/>
-                        <field name="user_id"/>
-                    </group>
-                </group>
-            </sheet>
-        </form>
-    </field>
-  </record>
+   <record model="ir.ui.view" id="record_form_view">
+       <field name="name">record.restricted.form</field>
+       <field name="model">record.restricted</field>
+       <field name="arch" type="xml">
+           <form>
+               <sheet>
+                   <group>
+                       <group>
+                           <field name="company_id" groups="base.group_multi_company"/>
```

Comment:
@AntoineVDV it's a bit sad to have a view with a only a poor little sad field :sob:

Reply:
Then add back some other fields but they should clearly be meaningless, like `name` or such.